cmake/scripts/ROOTConfig.cmake.in

```diff
@@ -78,6 +78,10 @@ endforeach()
 #----------------------------------------------------------------------------
 # Now set them to ROOT_LIBRARIES
 set(ROOT_LIBRARIES)
+if(MSVC)
```

Comment:
Should be `WIN32`, no?

Reply:
Not really, one can use gcc on Windows (with Cygwin or MingW), even if we don't officially support it...

Reply:
Ok, thanks for the clarification. I was thinking about LLVM...